src/social-network-provider/twitter.com/ui/fetch.ts

```diff
@@ -54,40 +57,57 @@ const registerUserCollector = () => {
         .then()
 }
 
+/** TODO: Dirty fix.
+ *   since MOW detected changes many times in post page,
+ *   we set an array here to storage posts and reduce.
+ */
+const keys: string[] = []
+
 const registerPostCollector = (self: SocialNetworkUI) => {
     new MutationObserverWatcher(postsSelectors())
         .useForeach((node, _, proxy) => {
-            const info = getEmptyPostInfo(postsRootSelector())
-            // push to map
-            self.posts.set(proxy, info)
-            const collectPostInfo = () => {
-                const r = postParser(node)
-                if (!r) return
+            const id = proxy.current.innerHTML
```

Comment:
You can use https://github.com/DimensionDev/Holoflows-Kit/blob/master/api-documents/kit.watcher.assignkeys.md this method to let the Watcher do the job for you.

Reply:
fixed 9807dd6d0b448c13dc1ef8898c72d887b312a5ed